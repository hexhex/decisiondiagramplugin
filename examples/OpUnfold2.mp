[common signature]
predicate: root/1;
predicate: innernode/1;
predicate: leafnode/2;
predicate: conditionaledge/5;
predicate: elseedge/2;

[belief base]
name: kb1;
mapping: "
	root(root).
	innernode(root).
	innernode(v1).
	innernode(v2).
	innernode(v3).
	leafnode(leaf1, class1).
	leafnode(leaf2, class2).
	conditionaledge(root, v1, x, \'<\', y).
	conditionaledge(root, v3, xx, \'<\', y).
	elseedge(root, v2).
	conditionaledge(v1, leaf1, z, \'<\', w).
	elseedge(v1, leaf2).
	conditionaledge(v2, leaf1, z, \'<\', w).
	elseedge(v2, leaf2).
	conditionaledge(v3, leaf1, zz, \'<\', w).
	elseedge(v3, leaf2).
";

[merging plan]
{
	operator: unfold;
	{
		kb1
	};
}
