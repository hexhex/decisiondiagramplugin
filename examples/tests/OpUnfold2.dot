digraph {
     v3 -> leaf2 [label="else"];
     v2 -> leaf2_2 [label="else"];
     v1 -> leaf2_1 [label="else"];
     root -> v2 [label="else"];
     v3 -> leaf1 [label="zz<w"];
     v2 -> leaf1_2 [label="z<w"];
     v1 -> leaf1_1 [label="z<w"];
     root -> v3 [label="xx<y"];
     root -> v1 [label="x<y"];
     leaf2_2 [label="leaf2 [class2]"];
     leaf1_1 [label="leaf1 [class1]"];
     leaf1_2 [label="leaf1 [class1]"];
     leaf2_1 [label="leaf2 [class2]"];
     leaf2 [label="leaf2 [class2]"];
     leaf1 [label="leaf1 [class1]"];
     v3 [label="v3"];
     v2 [label="v2"];
     v1 [label="v1"];
     root [label="root"];
}
