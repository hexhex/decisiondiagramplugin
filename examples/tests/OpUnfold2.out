{elseedge(v3,leaf2_1), elseedge(v2,leaf2), elseedge(v1,leaf2_2), elseedge(root,v2), conditionaledge(v3,leaf1_1,zz,"<",w), conditionaledge(v2,leaf1,z,"<",w), conditionaledge(v1,leaf1_2,z,"<",w), conditionaledge(root,v3,xx,"<",y), conditionaledge(root,v1,x,"<",y), leafnode(leaf1_2,class1), leafnode(leaf1_1,class1), leafnode(leaf2_2,class2), leafnode(leaf2_1,class2), leafnode(leaf2,class2), leafnode(leaf1,class1), innernode(v3), innernode(v2), innernode(v1), innernode(root), root(root)}
